js/ui/map.js

```diff
@@ -200,6 +203,7 @@ class Map extends Camera {
         if (options.style) this.setStyle(options.style);
 
         if (options.attributionControl) this.addControl(new AttributionControl());
+        if (options.logoControl) this.addControl(new LogoControl(), 'bottom-left');
```

Comment:
I think it's better to code the default position in the control code by adding a `getDefaultPosition` method.